libnmstate/netinfo.py

```diff
@@ -33,3 +35,13 @@ def show(*, include_status_data=False):
     """
     with plugin_context() as plugins:
         return show_with_plugins(plugins, include_status_data)
+
+
+def show_running_config():
```

Comment:
What is the difference between `show_running_config()` and `show()`?

Reply:
`show` is running state including auto state like DHCP/LLDP.

`show_running_config()` is configure only.